lib/src/util/customPropTypes.ts

```diff
@@ -49,3 +49,4 @@ export const optionalPercentNumber: PropTypes.Validator<number> =
     }
     return requiredPercentNumberValidator(props, propName, componentName);
   };
+
```

Comment:
Extraneous

Reply:
Without this newline, there is an error `Newline required at end of file but not found  eol-last`, I think we do need it based on the configuration `"eol-last": "error"`?

Reply:
> Without this newline, there is an error Newline required at end of file but not found eol-last, I think we do need it based on the configuration "eol-last": "error"?

The diff here is showing that the file ends in two newlines (the one between lines 51 and 52, and the one at the end of line 52).  It should end in one.

Reply:
> It also seems like this is in fact fixed? This file is no longer in the PR at all which was the goal.

There are two commits for this file, 
1dcfa28c3b4a8238c4dae01fe1b214f9270f621d I only saw one newline at the end of this file, and I removed it.
b179560e362379fc768750edde5b4d288c9d973f I realized the newline is required, then I added it back.

Reply:
> There are two commits for this file,
https://github.com/sonatype/sonatype-react-shared-components/commit/1dcfa28c3b4a8238c4dae01fe1b214f9270f621d I only saw one newline at the end of this file, and I removed it.
https://github.com/sonatype/sonatype-react-shared-components/commit/b179560e362379fc768750edde5b4d288c9d973f I realized the newline is required, then I added it back.

The important thing is that it's fixed now (it is), but if you want a deeper understanding, take a look at the way GH presents the diffs for those two commits.  They are not opposites.  The first one removed two newlines (resulting in a file which did not end in a newline and which GH calls out with the 🚫) while the second commit added back a single newline.